contracts/StablecoinConverter.sol

```diff
@@ -110,12 +110,15 @@ contract StablecoinConverter is EpochTokenLocker {
         return IdToAddressBiMap.getAddressAt(registeredTokens, id);
     }
 
+    uint public previousSolutionBatchId;
```

Comment:
Would it make sense to group these on a previousSolution struct?

Reply:
yeah...